Languages/IronPython/Tests/Inputs/UseCLI.py

```diff
@@ -13,11 +13,13 @@
 #
 #####################################################################################
 
-import clr
-clr.AddReference('System.Windows.Forms')
-from System.Windows.Forms import Form, Control
-
 from iptest.assert_util import *
+
+if not is_netstandard: # no System.Windows.Forms in netstandard
```

Comment:
It's used in conjunction with `test_nested_classes` in `test_clrload2.py`. It's already skipped for win32 and silverlight so I think we're good for now. Ideally we'd have a feature based system instead like @jdhardy suggested in #1366...